tsdb/points.go

```diff
@@ -707,7 +718,7 @@ func scanLine(buf []byte, i int) (int, []byte) {
 		}
 
 		// If we see a double quote, makes sure it is not escaped
-		if buf[i] == '"' && buf[i-1] != '\\' {
+		if buf[i] == '"' && (i-1 > 0 && buf[i-1] != '\\') {
```

Comment:
Basically, yes.  It prevents an index out of range panic when `"` is the first char.